src/components/ReviewList.js

```diff
@@ -1,48 +0,0 @@
-import React, { Component } from "react";
```

Comment:
This is just a duplicate file. The correct file is at 'src\components\**annotations**\ReviewList.js'